src/cijoe/cli/cli.py

```diff
@@ -351,27 +351,37 @@ def cli_workflow(args):
     return err
 
 
-def cli_interface(path):
-    path = Path(path)
-    args = parse_args()
-
+def create_tmp_workflow(args, paths):
     with tempfile.NamedTemporaryFile() as workflow:
         setattr(args, "workflow", Path(workflow.name))
+        setattr(args, "step", [])
```

Comment:
How about putting this into a jinja2 template? I think there is some general infra-structure in the code this.